packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -14,19 +14,22 @@ import 'text_selection_toolbar_button.dart';
 import 'theme.dart';
 
 // Values extracted from https://developer.apple.com/design/resources/.
-// The height of the toolbar, including the arrow.
-const double _kToolbarHeight = 45.0;
+
 // Vertical distance between the tip of the arrow and the line of text the arrow
 // is pointing to. The value used here is eyeballed.
 const double _kToolbarContentDistance = 8.0;
+
+// The size of the arrow button to navigate between pages.
 const Size _kToolbarArrowSize = Size(14.0, 7.0);
 
 // Minimal padding from tip of the selection toolbar arrow to horizontal edges of the
 // screen. Eyeballed value.
 const double _kArrowScreenPadding = 26.0;
 
-// Values extracted from https://developer.apple.com/design/resources/.
```

Comment:
Can you keep this comment or is there a reason to remove it?

Reply:
The same comment appears above - I separated it by one line so it's clearer it refers to all the constants.

Reply:
Many other constants here are eyeballed so I think it will be good to keep the provenance per constant, or per constant group.

Reply:
I'll keep comments per constant.